### ISSUE TITLE
Path to: `rascaline` release

### ISSUE THREAD

**Issue by jwa7** [Luthaf/rascaline#284]
Here we outline the steps needed to get to a rascaline release.

- [ ] Build steps, outlined in #281 
- [ ] Renaming metadata to match new conventions in metatensor
- [ ] Find a product name, free on PyPI, GitHub, [crate.io](https://crates.io/), no problematic related Google results

**Comment by Luthaf** [comment]
- [ ] Separate `cell` and `strain` gradients and fix cell gradients for small cell/large cutoff calculations

**Comment by jwa7** [comment]
- [ ] Agree on `k_{x}` numbering convention between the `cartesian_to_spherical` routine(s) and CG utils
- [ ] Generalised and re-factored CG tensor product routines, to be done in #316 
- [ ] Implement high level CG-based calculators for the main use cases, fully documented and tested

**Comment by Luthaf** [comment]
I don't think that anything related to the CG routines is related to the release. This issue is about stuff we can NOT do a release without, but CG can always be added/modified after an initial release (that's what incompatible version numbers are for). If you want to keep track of everything CG related somewhere, could you open a separate issue?
